sw/airborne/modules/gps/gps_ubx.c

```diff
@@ -380,6 +388,7 @@ static void gps_ubx_parse_nav_relposned(void)
       } else{
         gps_ubx.state.fix = 0;
       }
+    if(relPosValid) {
```

Comment:
Not sure if the fix state is valid in this case, but I guess it is ok. Though indentation seems off.

Reply:
Maybe I should revert this, it was a quick fix that I didn't use in the end. So if there is a chance that this is wrong, better remove it.